app/helpers/application_helper/toolbar/middleware_server_center.rb

```diff
@@ -37,4 +38,27 @@ class ApplicationHelper::Toolbar::MiddlewareServerCenter < ApplicationHelper::To
       ]
     ),
   ])
+  button_group('middleware_server_operations', [
+    select(
+      :middleware_server_power_choice,
+      'fa fa-power-off fa-lg',
+      t = N_('Power'),
+      t,
```

Comment:
I have no idea, but this is all over the place and I've just copied existing working solutions.